addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -38,6 +38,14 @@ def test_post_subscribe_recipients(self):
         self.assertEqual(self.test_record.message_follower_ids.mapped('partner_id'), original.mapped('partner_id') | self.partner_1 | self.partner_2)
         self.assertEqual(self.test_record.message_follower_ids.mapped('channel_id'), original.mapped('channel_id'))
 
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_chatter_mail_create_context_cleaning(self):
+        some_id = 2147483647
+        parent = self.env['res.partner'].create({'name': 'Parent'})
+        self.env.cr.execute(
```

Comment:
That's a bit strange actually :) is it required ? Can't we simply check that parent_id is False on the partner's creation message ?

Reply:
And/or use parent.id simply ?

Reply:
Yes, alright. I amended the commit with your suggestions.